scipy/linalg/flapack.pyf.src

```diff
@@ -887,6 +887,48 @@ interface
 
    end subroutine <prefix>getrf
 
+
+   subroutine <prefix2>sytrd(lower,n,a,lda,d,e,tau,work,lwork,info)
+     ! Reduce a real symmetric matrix A to real symmetric
+     ! tridiagonal form T by an orthogonal similarity transformation:
+     ! Q**T * A * Q = T.
+
+     callstatement (*f2py_func)((lower?"L":"U"),&n,a,&lda,d,e,tau,work,&lwork,&info);
+     callprotoargument char*,int*,<ctype2>*,int*,<ctype2>*,<ctype2>*,<ctype2>*,<ctype2>*,int*,int*
+
+     integer intent(hide),depend(a) :: lda=MAX(shape(a,0),1)
+     integer intent(hide),depend(a) :: n=shape(a,1)
+     integer optional,intent(in),check(lower==0||lower==1) :: lower = 0
+
+     <ftype2> dimension(lda,n),intent(in,out,copy,out=c) :: a
```

Comment:
This needs a squareness check before LAPACK chews it so it is best if we intervene 

    <ftype2> dimension(lda,n),check(shape(a,0)==shape(a,1)),intent(in,out,copy,out=c) :: a